Topic0/practiceTopic0/Practice4/Connection.java

```diff
@@ -0,0 +1,45 @@
+package practiceTopic0.Practice4;
+
+public class Connection {
+
+		private String user;
```

Comment:
let's say you create a connection this way:
Connection connection = new Connection();

Your code allows that but that connection isn't ready to be used, is it?
It has no data: password, db, user? What can you do to avoid this scenario?

Reply:
I should add those three arguments as inputs to this constructor. So that the class connection would have its fields user, pass and db with this data.